XML/Other/ReplaceableTestParameters.xml

```diff
@@ -687,7 +687,7 @@ Scenario 2 : You need to run all the performance tests quickly.
 	</Parameter>
 	<Parameter>
 		<ReplaceThis>HB_CUSTOM_KERNEL_BRANCH</ReplaceThis>
-		<ReplaceWith>master</ReplaceWith>
+		<ReplaceWith>stable</ReplaceWith>
```

Comment:
Why not set the value directly in the test case's xml file?

Reply:
When we started this test script, it was not either stable or master. So we took advantage of git branch system. I used master branch until the latest code breaking with call trace in SLES and RHEL. So, from my experiment, stable branch was ok. Dexuan and I agree to use stable branch.